downstream/modules/hub/proc-create-users.adoc

```diff
@@ -2,20 +2,20 @@
 // obtaining-token/master.adoc
 [id="proc-create-users"]
 
-= Creating a new user
+= Creating new users and giving them permissons
 
-You can create a user in {HubName} and add them to groups that can access features in the system associated by the level of assigned permissions.
+After you create a user in {HubName}, you can give them permissions by adding them to groups. Each group that can access features in the system associated to the level of assigned permissions.
 
 .Prerequisites
 
 * You have *user* permissions and can create users in {HubName}.
 
 .Procedure
 . Log in to your local {HubName}.
-. From the navigation panel, select menu:User Access[Users].
+. Navigate to menu:User Access[Users].
```

Comment:
We decided that the correct form for this is: "From the navigation panel, select menu:item1[]..."